build-go.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-export MAUTRIX_VERSION=$(cat go.mod | grep 'maunium.net/go/mautrix ' | awk '{ print $2 }')
```

Comment:
I continue to believe that it absolutely sucks Go doesn't let you alias this. It's just a source of endless merge conflicts :(

Reply:
It indeed makes some things much more difficult than they need to be :slightly_frowning_face:  I can say that I have attempted some form of aliasing in the past, but that only introduces other, arguably more confusing problems.